packages/react-native-web-examples/pages/image/index.js

```diff
@@ -114,6 +126,17 @@ export default function ImagePage() {
           />
         </View>
       </View>
+      <Divider />
+      <View style={styles.row}>
+        <View style={styles.column}>
+          <Text style={[styles.text]}>With Headers</Text>
+          <Image source={sourceWithHeaders} style={styles.image} />
+        </View>
+        <View style={styles.column}>
+          <Text style={[styles.text]}>Headers & Redirect</Text>
+          <Image source={sourceWithHeadersAndRedirect} style={styles.image} />
+        </View>
+      </View>
```

Comment:
Coming in a bit late here. But was curious if our goal with this to eventually merge this PR upstream into `react-native-web`? And if so, is that why we are also adding code to the "examples" package?

Asking mostly because we might want to keep the changes in this PR minimal for our fork so that conflicts are easier to resolve in the future but I have not worked much with this fork.

Reply:
We'll be trying to also publish this upstream to the original repo 

The examples package serves the purpose to visually verify the changes do what they are supposed to do 
It's an easy test to see if what you changed actually works and doesn't break existing use cases
`examples` can also be used in development with live reload
(I'm not sure if the examples are published anywhere at all, but just used locally)

Reply:
Got it, thanks. Mainly asking because it would be good to get a sense for what the maintainer's plans are and whether this is something they would merge or not and what their general expectations are when it comes to PRs - but I'll let @Beamanator follow up here.

Reply:
I think it makes sense to add examples to basically have visual tests for changes being made - if the maintainer doesn't like it they can ask us to remove, I can't see why they'd prefer _not_ to have examples & tests